extensions/wikia/Chat2/js/views/views.js

```diff
@@ -224,15 +225,16 @@ var UserView = Backbone.View.extend({
 		$(this.el).attr('id', this.liId());
 		$(this.el).attr('data-user', this.model.get('name'));
 
-		// If this is a chat moderator, add the chat-mod class so that kick-ban links don't show up, etc.
-		if(this.model.get('isModerator') === true){
-			$(this.el).addClass('chat-mod');
-		}
-
-		if(this.model.get('isStaff') === true){
-			$(this.el).addClass('staff');
-		}
+		// Check user groups so that kick-ban links don't show up, etc. & user gets a proper badge
+		$(this.el).addClass(groups.join(' '));
 
+		// if(this.model.get('isModerator') === true){
+		// 	$(this.el).addClass('chatmoderator');
+		// }
+		//
+		// if(groups.indexOf("staff") != -1){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)